src/middlewares/errorCatcher.js

```diff
@@ -0,0 +1,14 @@
+import status from '../config/status';
+
+const errorCatcher = controller => async (req, res, next) => {
```

Comment:
@luc-tuyishime give the function a more appropriate name

Reply:
Changes applied